crates/client/l1-messages/src/config.rs

```diff
@@ -0,0 +1,36 @@
+use std::fs::File;
+use std::path::PathBuf;
+
+pub const DEFAULT_ETHEREUM_NODE: &str = "127.0.0.1:8545";
+pub const DEFAULT_CONTRACT_ADDRESS: &str = "0x5fbdb2315678afecb367f032d93f642f64180aa3";
+
+#[derive(Clone, PartialEq, serde::Deserialize, Debug)]
+pub struct L1MessagesWorkerConfig {
+    #[serde(default = "default_http")]
+    pub http_provider: String,
+    #[serde(default = "default_contract_address")]
+    pub contract_address: String,
```

Comment:
the `ether` libs is already imported and expose an `Address` type.
https://docs.rs/ethers/latest/ethers/abi/type.Address.html

Reply:
For more security when people will be using non-default addresses it's probably better

Reply:
Sure!